lib/src/view/broadcast/broadcast_game_screen.dart

```diff
@@ -79,11 +81,15 @@ class _BroadcastGameScreenState extends ConsumerState<BroadcastGameScreen>
   Widget build(BuildContext context) {
     final broadcastGameState = ref
         .watch(broadcastGameControllerProvider(widget.roundId, widget.gameId));
+    final broadcastRoundState =
+        ref.watch(broadcastRoundControllerProvider(widget.roundId));
```

Comment:
I used `select` instead because `selectAsync` returns a `Future` which makes unpractical to use inside a widget.

Reply:
In order to use `selectAsync` you need to create a dedicated provider to retrieve the title. This is the way to do it with riverpod.

Reply:
But what is the advantage of creating a new provider instead of using `select` like this
```
final title = ref.watch(
      broadcastRoundControllerProvider(widget.roundId)
          .select((round) => round.value?.round.name),
    );
```

Reply:
It's mostly a problem of typing. Here you're mixing the async value which represent an asynchronous value which may be not here yet, with a nullable value that you derives from another asynchronous value.

By creating another provider that returns an AsyncValue<String> for the title you're making it clear of what you expect. It is ok to declare a new provider for that. It is like a cached computed value, but async. Doing that is recommended in the riverpod doc (but I don't remember where).

Reply:
Note also that using `.value?` directly you're not handling the error, it will be rethrown in the widget build method making it crash.

Reply:
Ok thanks for the detailed explanation. I'll make a title provider. 

I believe I have already done this in previous PR but I'll make sure to create a provider in the future